src/main/java/Decide.java

```diff
@@ -43,4 +43,44 @@ public static boolean LIC0(Point[] pts, Parameters params) {
         }
         return false;
     }
+
+    /**
+     * Checks distance between two points.
+     * @param pts array of Point objects
+     * @param params Parameter object that includes 
+     * q_pts, number of consecutive points, and QUADS, number of quadrants
+     * @return true if at least one set of Q_PTS consecutive data points that lie in more than QUADS quadrants.
```

Comment:
"true if at least one set of Q_PTS consecutive data points ~~that~~ lie in more than QUADS quadrants."?